ci-operator/config/openshift-priv/cluster-network-operator/openshift-priv-cluster-network-operator-master.yaml

```diff
@@ -65,13 +65,6 @@ tests:
   steps:
     cluster_profile: gcp
     workflow: openshift-upgrade-gcp-ovn
-- as: e2e-aws-sdn-upgrade
-  optional: true
-  steps:
-    cluster_profile: aws
-    env:
-      OVERRIDE_OPENSHIFT_SDN_DEPRECATION: "true"
-    workflow: openshift-upgrade-aws-sdn
```

Comment:
Aren't these z-stream upgrade jobs? We don't want to remove those while "master" is still 4.16

Reply:
Thanks @danwinship  for bringing this up. Let me hold this PR until 4.16 branching is created.